### ISSUE TITLE
Is it still working?

### ISSUE THREAD

**Issue by sengami** [alexschimpf/Snkrs-Bot#31]
Is there anyone can tell me is this still working??

**Comment by ericruiz-developer** [comment]
I don't believe it's working 100%, there has been changes to Nikes website and updates around bot detection. Not to speak on the owners behalf but I believe this is an open source project, intended for others to submit fixes or use as a base to spawn their own projects off of. 

You may need to have some programming knowledge and put in some work.

**Comment by sengami** [comment]
> I don't believe it's working 100%, there has been changes to Nikes website and updates around bot detection. Not to speak on the owners behalf but I believe this is an open source project, intended for others to submit fixes or use as a base to spawn their own projects off of.
> 
> You may need to have some programming knowledge and put in some work.

tks dude

**Comment by ValeFCF** [comment]
@ericruiz-developer can I run the script multiple times simultaneously?
